terraform/aks/workspace_variables/qa_aks.tfvars.json

```diff
@@ -0,0 +1,38 @@
+{
+  "paas_app_environment": "qa",
+  "azure_resource_prefix": "s189t01-att",
+  "key_vault_resource_group": "s189t01-att-qa-rg",
+  "key_vault_name": "s189t01-att-qa-kv",
+  "key_vault_app_secret_name": "APPLY-APP-SECRETS-QA",
+  "key_vault_infra_secret_name": "BAT-INFRA-SECRETS-QA",
+  "cluster": "test",
+  "namespace": "bat-qa",
+  "deploy_azure_backing_services": true,
+  "db_sslmode": "prefer",
```

Comment:
We should remove it to use the default value

Reply:
removed